javascript/net/grpc/web/grpcwebclientbase_test.js

```diff
@@ -75,6 +75,33 @@ testSuite({
     assertElementsEquals(DEFAULT_UNARY_HEADER_VALUES, Object.values(headers));
   },
 
+  async testRpcFalsyResponse() {
+    const xhr = new XhrIo();
+    const client = new GrpcWebClientBase(/* options= */ {}, xhr);
+    const methodDescriptor = createMethodDescriptor((bytes) => {
+      assertElementsEquals(DEFAULT_RPC_RESPONSE_DATA, [].slice.call(bytes));
+      return 0;
+    });
+
+    const response = await new Promise((resolve, reject) => {
+      client.rpcCall(
```

Comment:
Actually, after scratching my head a bit i realized that the main discussion we had was around "thenableCall" but it's not covered in tests here..

If it's not too difficult, i wonder if you could help add 1-2 thenableCall tests to cover the basic cases (e.g. normal response + false response)? Thanks a lot!!

Reply:
ya let me try

Reply:
added

Reply:
wow thanks so much for the new tests. Looks awesome! 😃